hack/common.mk

```diff
@@ -79,6 +79,9 @@ endif
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+gen-mocks: mockgen
+	PATH=$(PATH):$(shell pwd)/bin $(GO) generate $(V_FLAG) ./...
```

Comment:
I would put `bin` folder as the first thing in the path, so that we can be sure the tight `mockgen` version is used.

Reply:
Fixed.